akash1/HTML/ASSIn5/test1.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+         <meta charset="UTF-8">
+         <meta http-equiv="X-UA-Compatible" content="IE=edge">
+         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+         <title>webpage</title>
+         <link rel="stylesheet" href="style1.css">
+         <link href="https://fonts.googleapis.com/css2?family=Baloo+Bhaina+2&display=swap" rel="stylesheet">
+   </head>
+    <body>
+        <div class="first">
+            <div class="second"><strong>Akash Kadam</strong></div>
+        </div>
```

Comment:
remove a closing div tag.

Reply:
i have to use two  <div> tag
that is tag inside tag so why i have to remove that tag